src/pyodide/BUILD.bazel

```diff
@@ -1,8 +1,9 @@
 load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
-load("@bazel_skylib//rules:write_file.bzl", "write_file")
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
 load("@capnp-cpp//src/capnp:cc_capnp_library.bzl", "cc_capnp_library")
 load("//:build/capnp_embed.bzl", "capnp_embed")
+load("//:build/pyodide_bucket.bzl", "PYODIDE_PACKAGE_BUCKET_URL")
```

Comment:
Lots of unrelated changes here but they do make the file nicer.

Reply:
This is buildifier fixing the formatting errors.